doc/weakly_supervised.rst

```diff
@@ -144,18 +151,23 @@ Or alternatively (using a preprocessor):
 >>> mmc = MMC(preprocessor=X, random_state=42)
 >>> mmc.fit(pairs_indice, y)
 
-
 Now that the estimator is fitted, you can use it on new data for several
 purposes.
 
-First, you can transform the data in the learned space, using `transform`:
-Here we transform two points in the new embedding space.
+First, your mahalanobis learner can transform the data in
+the learned space, using `transform`: Here we transform two points in
+the new embedding space.
 
 >>> X_new = np.array([[9.4, 4.1, 4.2], [2.1, 4.4, 2.3]])
 >>> mmc.transform(X_new)
 array([[-3.24667162e+01,  4.62622348e-07,  3.88325421e-08],
        [-3.61531114e+01,  4.86778289e-07,  2.12654397e-08]])
 
+.. warning::
+    
+    If you try to use `transform` with a similarity learner, an error will
+    appear, as you cannot transform the data using them.
+
 Also, as explained before, our metric learner has learned a distance between
```

Comment:
re-organize as for supervised learners

Reply:
Reorganized just like supervised learners